### PR TITLE
[bitnami/postgresql] fix annotation typo

### PR DESCRIPTION
**Pull request by jcpunk**
### Description of the change

Correctly check for the annotations to be used

### Benefits

Correct use of annotations on Jobs

### Applicable issues

- fixes #

### Additional information

Noted in https://github.com/bitnami/charts/pull/17852#discussion_r1297325693

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/postgresql/templates/backup/cronjob.yaml [hunk 1]
```diff
@@ -50,7 +50,7 @@ spec:
             {{- if .Values.commonLabels }}
             {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 12 }}
             {{- end }}
-          {{- if or .Values.annotations .Values.commonAnnotations }}
+          {{- if or .Values.backup.cronjon.annotations .Values.commonAnnotations }}
```
Comment (migruiz4) [inline bitnami/postgresql/templates/backup/cronjob.yaml:53]: Hi @jcpunk,

There is a typo here:
```suggestion
          {{- if or .Values.backup.cronjob.annotations .Values.commonAnnotations }}

File: bitnami/postgresql/Chart.yaml [hunk 2]
```diff
@@ -35,4 +35,4 @@ maintainers:
 name: postgresql
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/postgresql
-version: 12.8.2
+version: 12.8.3
```
Comment (migruiz4) [inline bitnami/postgresql/Chart.yaml:38]: Version `12.8.3` has been already released but Github UI does not display it, could you please bump an additional patch version? (12.8.4)

**Comment by jcpunk** [comment]
In theory I've got this sorted.

**Review by migruiz4 — APPROVED** [review]
Thank you for your contribution, looks good to me!

**Pull request merged**